src/main/java/io/druid/data/input/impl/RegexParseSpec.java

```diff
@@ -39,13 +41,18 @@ public RegexParseSpec(
       @JsonProperty("timestampSpec") TimestampSpec timestampSpec,
       @JsonProperty("dimensionsSpec") DimensionsSpec dimensionsSpec,
       @JsonProperty("listDelimiter") String listDelimiter,
+      @JsonProperty("columns") List<String> columns,
       @JsonProperty("pattern") String pattern
   )
   {
     super(timestampSpec, dimensionsSpec);
 
     this.listDelimiter = listDelimiter;
+    Preconditions.checkNotNull(columns, "columns");
```

Comment:
my bad, actually csv/tsv also expec columns to be non-null , parsers consider them to be non-nullable.